app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -83,9 +83,19 @@ function UnknownError({
     );
   }
   if ((isFailedSelfieLivenessOrQuality || isFailedSelfie) && err) {
+    let selfieHelpCenterLink = helpCenterLink;
+    let selfieHelpCenterLinkText = t('doc_auth.errors.general.selfie_failure_help_link_text');
+    if (isFailedSelfieLivenessOrQuality) {
+      selfieHelpCenterLink += '#anchor'
```

Comment:
WDYT of using a `URL` object to parse + manipulate this?
```suggestion
      const helpCenterURL = new URL(helpCenterLink)
      helpCenterURL.hash = 'anchor';
      selfieHelpCenterLink = helpCenterURL.toString();
```

Reply:
We also have a similar abstraction [`addSearchParams`](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/url/add-search-params.ts). I could imagine abstracting a similar `addHashAnchor` or generalized `addParams` if we wanted to have a simple one-liner tool available, if we think this is a common pattern.

Reply:
Another thought, having this URL be an anchor is maybe just an implementation detail? What if both URLs were passed from the server as configs? Simplifies the frontend logic